### PR TITLE
binarysearch.cpp

### PR DESCRIPTION
**Pull request by Sshruti10**
Binary Search in C++.

### CODE REVIEW

**Review by 22shubh22 — COMMENTED** [review]
Check for suggestions in the code.

File: search-algos/binary_search/binarysearch.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
+include<iostream.h>
```
Comment (22shubh22) [inline search-algos/binary_search/binarysearch.cpp]: not a valid header file.

File: search-algos/binary_search/binarysearch.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+int Bsearch(int[],int,int);         //Bsearch(array,size,item to be searched)
+int main()
+{
+int A[50],item,n,index;
```
Comment (22shubh22) [inline search-algos/binary_search/binarysearch.cpp]: first declare the array size. So the capacity of array will not be restricted.

File: search-algos/binary_search/binarysearch.cpp [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+cout<<"Enter desired array size(max. 50)";
+cin>>n;
+cout<<"\nEnter array elements in ascending order";
+for(int i=0;i<n;i++)
+ {
+    cin>>A[i];
```
Comment (22shubh22) [inline search-algos/binary_search/binarysearch.cpp:12]: add standard namespace file.

**Review by 22shubh22 — COMMENTED** [review]
Check for suggestions in the code.

**Review by khanchi97 — CHANGES_REQUESTED** [review]
Please make required changes.

**Review by khanchi97 — APPROVED** [review]

**Pull request merged**